Bio/PDB/PDBList.py

```diff
@@ -701,88 +702,166 @@ def get_seqres_file(self, savefile="pdb_seqres.txt"):
     the last will be considered). By default (no format specified) structures are
     downloaded as PDBx/mmCif files.
     """
-    print(doc)
-
-    file_format = "mmCif"
-    overwrite = False
-    with_assemblies = False
-
-    if len(sys.argv) > 2:
-        pdb_path = sys.argv[2]
-        pl = PDBList(pdb=pdb_path)
-        if len(sys.argv) > 3:
-            for option in sys.argv[3:]:
-                if option == "-d":
-                    pl.flat_tree = True
-                elif option == "-o":
-                    overwrite = True
-                elif option in ("-pdb", "-xml", "-mmtf"):
-                    file_format = option[1:]
-                # Allow for download of assemblies alongside ASU
-                elif option == "-with-assemblies":
-                    with_assemblies = True
 
-    else:
-        pdb_path = os.getcwd()
-        pl = PDBList()
-        pl.flat_tree = True
-
-    if len(sys.argv) > 1:
-        if sys.argv[1] == "update":
-            # update PDB
-            print("updating local PDB at " + pdb_path)
-            pl.update_pdb(file_format=file_format, with_assemblies=with_assemblies)
+    desc = """
+    (c) Kristian Rother 2003, Wiktoria Karwicka & Jacek Smietanski 2016
+    Contributed to Biopython
 
-        elif sys.argv[1] == "all":
-            # get the entire PDB
-            pl.download_entire_pdb(file_format=file_format)
-            if with_assemblies:
-                # get all assembly structures
-                pl.download_all_assemblies(file_format=file_format)
+    Quick access to the structure lists on the PDB or its mirrors.
+    Requires a mode (update/all/obsol/assemb/single/multi) to be specified.
+    By default (no format specified) structures are downloaded as PDBx/mmCif files.
+    """
 
-        elif sys.argv[1] == "obsol":
-            # get all obsolete entries
-            pl.download_obsolete_entries(pdb_path, file_format=file_format)
+    # Instantiate argument parser/subparser
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter, description=desc
+    )
+
+    # Add positional argument for pdb_path
+    parser.add_argument(
+        "pdb_path",
+        nargs="?",
+        default=os.getcwd(),
+        help="local path to use, defaults to the current working directory",
```

Comment:
Nitpick: we can follow the grammar, question of style, the important is to be consistent 
```suggestion
        help="Local path to use, defaults to the current working directory.",
```

Reply:
I generally like the look of all lower case in the help messages that I get and this was consistent with the style of the original docstring that currently prints when PDBList.py is called directly from the CLI.

Reply:
:+1: